tools/mypy-onnx.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+import subprocess
+import os
+
+
+def main():
+    try:
+        subprocess.check_call(["mypy", "."])
+        subprocess.check_call(["mypy", "--py2", "."])
```

Comment:
use __file__ and some path resolution functions to get the top level directory path

Reply:
I moved it below the `chdir`, that should fix it